cloud/mock.go

```diff
@@ -240,10 +240,10 @@ func (mockMgr *mockManager) TimeTilNextPayment(host *host.Host) time.Duration {
 }
 
 func (mockMgr *mockManager) GetInstanceStatuses(ctx context.Context, hosts []host.Host) ([]CloudStatus, error) {
-	if len(hosts) != 2 {
-		return nil, errors.New("expecting 2 hosts")
+	if len(hosts) != 1 {
```

Comment:
Why did this test change?

Reply:
Each call to GetInstanceStatuses from the mockManager should now see one host, because the HostStatus job now creates a different manager for each region for a given provider. Previously there were 2 hosts with the same provider, but now they each have different regions and should therefore each have their own managers and GetInstanceStatuses calls.